03-Raster-Calculations-In-R.Rmd

```diff
@@ -42,10 +43,16 @@ raster math.
 * Know how to perform a more efficient subtraction (difference) between two 
 rasters using the raster `overlay()` function in R.
 
+###Challenge Code
```

Comment:
The more i see this, the more i think we can have an "about these lessons" page and just link to it.  The link might say, lesson format, challenge answer...